spaces/urls.py

```diff
@@ -1,8 +1,9 @@
 from django.urls import path
 
-from spaces.views import SpaceListView, ReviewView
+from spaces.views import SpaceListView, ReviewView, SpaceRegisterView
 
 urlpatterns = [
     path('', SpaceListView.as_view()),
     path('/reviews', ReviewView.as_view()),
+    path('/register', SpaceRegisterView.as_view()),
```

Comment:
SpaceRegisterView의 post handler method를 SpaceListView로 옮겨주세요.
```suggestion
    path('/register', SpaceRegisterView.as_view()),

        POST :8000/spaces/register
        POST :8000/spaces
```